src/taskapi/task.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package taskapi;
```

Comment:
I don't think the api part is really needed. I think we should just keep the package as task and disambiguate with import renames if we need both `src/task` and `src/internal/task`.

Reply:
unfortunately proto seems to need `protoPackage.messageName` to be unique, so this can't just be `task` unless I also change some of the messages. I can, however, change the go package to just `task`